multi-repo-action/org-workflow-add.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/google/go-github/v42/github"
+	"golang.org/x/oauth2"
+)
+
+// *****SET THESE PARAMETERS*****
+const ORG_NAME string = "ORGANIZATION NAME"
+const PAT string = "PERSONAL ACCESS TOKEN"
+
+var REPO_LIST = []string{} // OPTIONAL, LEAVE EMPTY TO PROCESS ALL REPOS UNDER ORG
+
+// Adds the OpenSSF Scorecard workflow to all repositores under the given organization.
+func main() {
+	// Get github user client.
+	context := context.Background()
+	tokenService := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: PAT},
+	)
+	tokenClient := oauth2.NewClient(context, tokenService)
+	client := github.NewClient(tokenClient)
+
+	// If not provided, get all repositories under organization.
+	if len(REPO_LIST) == 0 {
```

Comment:
if REPO_LIST is provided by user, we should sanity-check that the org contains all these repos.